### DISCUSSION TITLE
Annotations API - GET Method

### DISCUSSION THREAD

**Discussion by huy7070** [grafana/grafana#47521] [General]
Hello. I was wondering why `GET` for `/api/annotations/:id` is not available? I would like to get information about one annotation via its `id`. Thanks.

**Comment by papagian** [comment]
[There is](https://github.com/grafana/grafana/blob/c56aae6f63ce68e0ea11041112c5b4f7ffd0d7eb/pkg/api/api.go#L525) but unfortunately it's missing from the docs (it's defined [in swagger](https://github.com/grafana/grafana/blob/c56aae6f63ce68e0ea11041112c5b4f7ffd0d7eb/public/api-merged.json#L2181) though); I will make sure that we will update the docs appropriately.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
